### DISCUSSION TITLE
Add support for Swift packages vulnerabilities

### DISCUSSION THREAD

**Discussion by naortalmor1** [aquasecurity/trivy#4895] [Ideas]
### Description

Github advisory now contain swift packages vulnerabilities - https://github.com/advisories?query=type%3Areviewed+ecosystem%3Aswift

Would love to add this capability to trivy

### Target

Filesystem

### Scanner

Vulnerability

**Comment by knqyf263** [comment]
Tracking in https://github.com/aquasecurity/trivy/issues/4896
